packages/server/api.js

```diff
@@ -384,39 +384,6 @@ module.exports = async function attachAPI(app, {wss, db, dbDir}) {
     }
   ])
 
-  app.post('/api/pin-message', [
-    ...middleware.loadVarFromBody('messageID'),
-    ...middleware.getSessionUserFromID('sessionID', 'sessionUser'),
-    ...middleware.requireBeAdmin('sessionUser'),
-    ...middleware.getMessageFromID('messageID', 'message'),
-    (req, res, next) => {
-      const v = req[middleware.vars]
-      v.channelID = v.message.channelID
-      next()
-    },
-    ...middleware.getChannelFromID('channelID', 'channel'),
-
-    async (request, response) => {
-      const { messageID, channel } = request[middleware.vars]
-
-      if (channel.pinnedMessageIDs.includes(messageID)) {
-        response.status(500).json({
-          error: errors.ALREADY_PERFORMED
-        })
-
-        return
-      }
-
-      await db.channels.update({_id: channel._id}, {
-        $push: {
-          pinnedMessageIDs: messageID
-        }
-      })
-
-      response.status(200).json({})
-    }
-  ])
-
   app.post('/api/add-message-reaction', [
```

Comment:
Oh yeah, still need to do reactions. I'm thinking `POST /api/messages/reactions` to create, etc.?

Reply:
Yes, but honestly I'm not worrying much about reactions right now. They 100% do not work with the emote system yet. So don't bother documenting reactions, either.